Waters/SunkenSeaWater.cs

```diff
@@ -1,33 +1,58 @@
-﻿using Microsoft.Xna.Framework;
+﻿using CalamityMod.Particles;
+using System;
+using CalamityMod.Systems;
+using CalamityMod.Tiles.Abyss;
+using Microsoft.Xna.Framework;
+using Terraria;
 using Terraria.ModLoader;
+using Terraria.Graphics;
+using CalamityMod.Dusts.WaterSplash;
+using CalamityMod.Gores.WaterDroplet;
 
 namespace CalamityMod.Waters
 {
-    public class SunkenSeaWater : ModWaterStyle
-    {
-        public static int Type;
-        public override void SetStaticDefaults()
-        {
-            Type = Slot;
-        }
-        public override int ChooseWaterfallStyle()
-        {
-            return ModContent.Find<ModWaterfallStyle>("CalamityMod/SunkenSeaWaterflow").Slot;
-        }
+    public class SunkenSeaWaterflow : ModWaterfallStyle { }
 
-        public override int GetSplashDust()
-        {
-            return 33;
-        }
-
-        public override int GetDropletGore()
+    public class SunkenSeaWater : CalamityModWaterStyle
+    {
+        public override int ChooseWaterfallStyle() => ModContent.Find<ModWaterfallStyle>("CalamityMod/SunkenSeaWaterflow").Slot;
+        public override int GetSplashDust() => ModContent.DustType<SunkenSeaSplash>();
+        public override int GetDropletGore() => ModContent.GoreType<SunkenSeaWaterDroplet>();
+        public override Color BiomeHairColor() => new Color(46, 155, 1716);
```

Comment:
Its been 12 hours since I saw this comment and im still laughing. I have 0 clue on how it ended up being ``1716``. I've corrected this to be ``171`` instead.